cluster/images/etcd/Makefile

```diff
@@ -50,10 +50,20 @@ ifeq ($(ARCH),s390x)
 	BASEIMAGE?=s390x/busybox
 endif
 
+# Use GNU cp
+if LANG=C cp --help 2>&1 | grep -q GNU; then
```

Comment:
this is a Makefile, not a shell script, so I don't think these conditionals work?